literature.bib

```diff
@@ -10691,15 +10694,15 @@ @article{MKB:JSME09
 }
 
 @article{BDGG:NC09,
-	author={Bianchi, Leonora and Dorigo, Marco and Gambardella, LucaMaria and Gutjahr, WalterJ.},
-	title={{A Survey on Metaheuristics for Stochastic Combinatorial Optimization}},
-	issn={1567-7818},
-	journal={Natural Computing},
-	volume={8},
-	number={2},
-	publisher={Springer Netherlands},
-	pages={239-287},
-	year=2009
+	author = {Bianchi, Leonora and Dorigo, Marco and Gambardella, LucaMaria and Gutjahr, WalterJ.},
+	title = {{A Survey on Metaheuristics for Stochastic Combinatorial Optimization}},
+	issn = {1567-7818},
+	journal = {Natural Computing},
```

Comment:
same here

Reply:
Not mine, just fixed the spaces.

Reply:
https://github.com/tthuem/BibTags/blame/master/literature.bib#L10468